### DISCUSSION TITLE
DaisyUI + Ruby on Rails 7 with Tailwind activated

### DISCUSSION THREAD

**Discussion by gobijan** [saadeghi/daisyui#580] [Q&A]
Hey,

Rails 7 comes with tailwindcss as an out of the box option using a gem that wraps around the official tailwind binary. 
What is the best way to use DaisyUI on a machine that has no node installed. So far I got it working using the CDN but it's not recommended for PROD right?
Shall one download the whole framework and use it via asset pipeline? 

Any thoughts?

Cheers

**Comment by saadeghi** [comment]
As far as I know, without Node.js, only Tailwind's official plugins are supported (Typography, etc...)  
The problem with using CDN on production is that it includes every responsive variant of class names (whether is used in HTML or not). Currently It's 212kb (29kb compressed).
I have no experience in Rails but there might be PurgeCSS or similar packages for it to remove unused styles (maybe [this](https://github.com/sebyx07/purgecss_rails)?)
